### ISSUE TITLE
Issues with python scripts/main.py

### ISSUE THREAD

**Issue by crowlsyong** [Torantulino/Auto-GPT#490]
### Duplicates

- [X] I have searched the existing issues

### Steps to reproduce 🕹

- I added OpenAI API Key in Windows Environment Variables
- In Powershell _without_ Admin

1. `git clone https://github.com/Torantulino/Auto-GPT.git`
2.  `cd 'Auto-GPT'`
3. `pip install -r requirements.txt`
4. Add Pinecone and OpenAI API keys to `.env` file
5. `python scripts/main.py`

**System Info**:
Edition	Windows 11 Pro
Version	22H2
Installed on	‎1/‎10/‎2023
OS build	22621.1413

**System Info Cont'd**:
Experience	Windows Feature Experience Pack 1000.22639.1000.0
Processor	AMD Ryzen 7 3700X 8-Core Processor                3.60 GHz
Installed RAM	32.0 GB
System type	64-bit operating system, x64-based processor
Pen and touch	No pen or touch input is available for this display

### Current behavior 😯

Welcome to Auto-GPT!  Enter the name of your AI and its role below. Entering nothing will load defaults.
Name your AI:  For example, 'Entrepreneur-GPT'
AI Name: Supportive-GPT
Supportive-GPT here! I am at your service.
Describe your AI's role:  For example, 'an AI designed to autonomously develop and run businesses with the sole goal of increasing your net worth.'
Supportive-GPT is: an AI designed to autonomously develop and run businesses with the sole goal of increasing your net worth
Enter up to 5 goals for your AI:  For example: Increase net worth, Grow Twitter Account, Develop and manage multiple businesses autonomously'
Enter nothing to load defaults, enter nothing when finished.
Goal 1: Increase net worth
Goal 2: Increase followers
Goal 3: Increase health and wellbeing
Goal 4: Increase time spent out doors
Goal 5: Increase health and well being of others
Traceback (most recent call last):
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\urllib3\connection.py", line 174, in _new_conn
    conn = connection.create_connection(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\urllib3\util\connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python311\Lib\socket.py", line 962, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
socket.gaierror: [Errno 11001] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\urllib3\connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
                       ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\urllib3\connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\urllib3\connectionpool.py", line 1042, in _validate_conn
    conn.connect()
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\urllib3\connection.py", line 363, in connect
    self.sock = conn = self._new_conn()
                       ^^^^^^^^^^^^^^^^
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\urllib3\connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x000001B9BF233310>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\crowl\Documents\Code\AI\Auto-GPT\scripts\main.py", line 286, in <module>
    memory = PineconeMemory()
             ^^^^^^^^^^^^^^^^
  File "C:\Users\crowl\Documents\Code\AI\Auto-GPT\scripts\config.py", line 17, in __call__
    cls._instances[cls] = super(
                          ^^^^^^
  File "C:\Users\crowl\Documents\Code\AI\Auto-GPT\scripts\memory.py", line 30, in __init__
    if table_name not in pinecone.list_indexes():
                         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\pinecone\manage.py", line 185, in list_indexes
    response = api_instance.list_indexes()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\pinecone\core\client\api_client.py", line 776, in __call__
    return self.callable(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\pinecone\core\client\api\index_operations_api.py", line 1132, in __list_indexes
    return self.call_with_http_info(**kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\pinecone\core\client\api_client.py", line 838, in call_with_http_info
    return self.api_client.call_api(
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\pinecone\core\client\api_client.py", line 413, in call_api
    return self.__call_api(resource_path, method,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\pinecone\core\client\api_client.py", line 200, in __call_api
    response_data = self.request(
                    ^^^^^^^^^^^^^
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\pinecone\core\client\api_client.py", line 439, in request
    return self.rest_client.GET(url,
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\pinecone\core\client\rest.py", line 236, in GET
    return self.request("GET", url,
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\pinecone\core\client\rest.py", line 202, in request
    r = self.pool_manager.request(method, url,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\urllib3\request.py", line 74, in request
    return self.request_encode_url(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\urllib3\request.py", line 96, in request_encode_url
    return self.urlopen(method, url, **extra_kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\urllib3\poolmanager.py", line 376, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\urllib3\connectionpool.py", line 815, in urlopen
    return self.urlopen(
           ^^^^^^^^^^^^^
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\urllib3\connectionpool.py", line 815, in urlopen
    return self.urlopen(
           ^^^^^^^^^^^^^
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\urllib3\connectionpool.py", line 815, in urlopen
    return self.urlopen(
           ^^^^^^^^^^^^^
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\urllib3\connectionpool.py", line 787, in urlopen
    retries = retries.increment(
              ^^^^^^^^^^^^^^^^^^
  File "C:\Users\crowl\AppData\Roaming\Python\Python311\site-packages\urllib3\util\retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='controller.9ae2e38c-b839-4055-9810-f651bfdbd5d8.pinecone.io', port=443): Max retries exceeded with url: /databases (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001B9BF233310>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))
PS C:\Users\crowl\Documents\Code\AI\Auto-GPT>

### Expected behavior 🤔

I expect to be able to get past the 'Goals' section of the AI. 

### Your prompt 📝

```yaml
# NOT APPLICABLE
```

**Comment by richbeales** [comment]
The first time the app runs it has to create the pinecone database, which can take a couple of minutes.  If you run it again does it work now?

**Comment by crowlsyong** [comment]
@richbeales 
I think I found the issue, it was my mistake in the `.env` file:
![image](https://user-images.githubusercontent.com/81304661/230725246-88d6b294-aed2-42c0-b7fa-810be1ab2b81.png)

After updating my `.env` to not have that silly mistake, I no longer getting the error. It seems to be building now

I am now awaiting the next step. It appears to be building right now. I'll report back if I am able to proceed from here:
![image](https://user-images.githubusercontent.com/81304661/230725513-f84df759-df7b-4581-8ffa-3fc9ba69edd4.png)

Thank you for the help. Sorry for being a klutz.

**Comment by crowlsyong** [comment]
@richbeales ✅Success!
**Note**- your original comment still holds true- the first time I ran it, it just kinda sat there (without errors). After a few minutes, I exited and ran it again and it was succesful:
![image](https://user-images.githubusercontent.com/81304661/230726461-3bb524e9-d921-422f-9ee2-a2d2f1b571fc.png)
P.S. Look at the last prompt- I told it to take breaks and so it suggested that it shut itself off after my first few prompts. This is amazing.
